LayoutTests/fast/table/border-changes-expected.txt

```diff
@@ -67,10 +67,10 @@ PASS getComputedStyle(cell, '').borderLeftWidth is '1px'
 PASS getComputedStyle(cell, '').borderRightWidth is '1px'
 PASS getComputedStyle(cell, '').borderTopWidth is '1px'
 PASS getComputedStyle(cell, '').borderBottomWidth is '1px'
-PASS getComputedStyle(cell, '').borderLeftStyle is 'solid'
-PASS getComputedStyle(cell, '').borderRightStyle is 'solid'
-PASS getComputedStyle(cell, '').borderTopStyle is 'solid'
-PASS getComputedStyle(cell, '').borderBottomStyle is 'solid'
+FAIL getComputedStyle(cell, '').borderLeftStyle should be inset. Was solid.
+FAIL getComputedStyle(cell, '').borderRightStyle should be inset. Was solid.
+FAIL getComputedStyle(cell, '').borderTopStyle should be inset. Was solid.
+FAIL getComputedStyle(cell, '').borderBottomStyle should be inset. Was solid.
```

Comment:
FAIL messages are not OK, we should change the test if needed.

Reply:
Thanks @nt1m 
The test is good but WebKit fails for now.

This is STP 170 and Firefox side by side.
![Screenshot 2023-05-25 at 12 25 12](https://github.com/WebKit/WebKit/assets/505230/3759dc9e-b358-42f8-9dd6-65aae9cc76f8)

The Patch fixes some of the fails related to the report of Daniel Holbert. I discovered more secondary issue. 
And it's why I opened the followup bug https://bugs.webkit.org/show_bug.cgi?id=257254

What would you recommend?

Reply:
Ah ok, I guess we can keep it failing and fix it later. Though ideally, we'd change and fix the test at the same time.